README.md

```diff
@@ -38,6 +38,12 @@ git submodule update --init --recursive
 
 Then set up [git secrets](#git-secrets) and fire up the [development servers](#running-the-dev-servers). Optionally, you can [set up your Intellij](https://docs.google.com/document/d/1DtESBapEzvuti7xODTFPHorwmLM7LybF-6D5lhbIkLU/edit) for UI or API work.
 
+Before doing any development, you must run:
+```Shell
+./project.rb appengineRun
+```
+This will generate MapStruct files that are necessary for the app to compile. This command will never complete - when it has gotten to 97% it will hang forever. It can safely be `ctrl+c`'d at that point.
```

Comment:
'97%' is apparently Mac-only: https://precisionmedicineinitiative.atlassian.net/browse/RW-3446